Makefile

```diff
@@ -229,6 +237,11 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 		make all install; \
 	fi
 
+.install.varlink: .gopathok
+	$(GO) get -u github.com/varlink/go/varlink
+	$(GO) get -u github.com/varlink/go/cmd/varlink-go-interface-generator
+	$(GO) generate ./cmd/podman/ioprojectatomicpodman/...
```

Comment:
This really ought to be separate from install.varlink - generate.varlink target, or just put it in make, localintegration, localunit as needed

Reply:
Ok done